Failed to rebuild/clean install on Atom v1.2.0

Get an error while trying to install this package on most recent version of Atom (1.2.0) Working well with Atom 1.1.x.
```bash
➜  apm install recent-files-fuzzy-finder                                      
Installing recent-files-fuzzy-finder to /home/vtikhonov/.atom/packages ✗

> runas@2.0.0 install /tmp/apm-install-dir-1151016-25623-bm7lg8/node_modules/recent-files-fuzzy-finder/node_modules/pathwatcher/node_modules/runas
> node-gyp rebuild

make: Entering directory `/tmp/apm-install-dir-1151016-25623-bm7lg8/node_modules/recent-files-fuzzy-finder/node_modules/pathwatcher/node_modules/runas/build'
  CXX(target) Release/obj.target/runas/src/main.o
make: Leaving directory `/tmp/apm-install-dir-1151016-25623-bm7lg8/node_modules/recent-files-fuzzy-finder/node_modules/pathwatcher/node_modules/runas/build'

In file included from ../src/main.cc:1:0:
../../nan/nan.h:260:25: error: redefinition of ‘template<class T> v8::Local<T> _NanEnsureLocal(v8::Local<T>)’
 NAN_INLINE v8::Local<T> _NanEnsureLocal(v8::Local<T> val) {
                         ^
../../nan/nan.h:255:25: error: ‘template<class T> v8::Local<T> _NanEnsureLocal(v8::Handle<T>)’ previously declared here
 NAN_INLINE v8::Local<T> _NanEnsureLocal(v8::Handle<T> val) {
                         ^
../../nan/nan.h:660:13: error: ‘node::smalloc’ has not been declared
     , node::smalloc::FreeCallback callback
             ^
../../nan/nan.h:660:35: error: expected ‘,’ or ‘...’ before ‘callback’
     , node::smalloc::FreeCallback callback
                                   ^
../../nan/nan.h: In function ‘v8::Local<v8::Object> NanNewBufferHandle(char*, size_t, int)’:
../../nan/nan.h:664:50: error: ‘callback’ was not declared in this scope
         v8::Isolate::GetCurrent(), data, length, callback, hint);
                                                  ^
../../nan/nan.h:664:60: error: ‘hint’ was not declared in this scope
         v8::Isolate::GetCurrent(), data, length, callback, hint);
                                                            ^
../../nan/nan.h: In function ‘v8::Local<v8::Object> NanNewBufferHandle(const char*, uint32_t)’:
../../nan/nan.h:671:67: error: call of overloaded ‘New(v8::Isolate*, const char*&, uint32_t&)’ is ambiguous
     return node::Buffer::New(v8::Isolate::GetCurrent(), data, size);
                                                                   ^
../../nan/nan.h:671:67: note: candidates are:
In file included from ../../nan/nan.h:25:0,
                 from ../src/main.cc:1:
/home/vtikhonov/.atom/.node-gyp/.node-gyp/0.34.0/src/node_buffer.h:41:40: note: v8::MaybeLocal<v8::Object> node::Buffer::New(v8::Isolate*, v8::Local<v8::String>, node::encoding) <near match>
 NODE_EXTERN v8::MaybeLocal<v8::Object> New(v8::Isolate* isolate,
                                        ^
/home/vtikhonov/.atom/.node-gyp/.node-gyp/0.34.0/src/node_buffer.h:41:40: note:   no known conversion for argument 3 from ‘uint32_t {aka unsigned int}’ to ‘node::encoding’
/home/vtikhonov/.atom/.node-gyp/.node-gyp/0.34.0/src/node_buffer.h:53:40: note: v8::MaybeLocal<v8::Object> node::Buffer::New(v8::Isolate*, char*, size_t) <near match>
 NODE_EXTERN v8::MaybeLocal<v8::Object> New(v8::Isolate* isolate,
                                        ^
/home/vtikhonov/.atom/.node-gyp/.node-gyp/0.34.0/src/node_buffer.h:53:40: note:   no known conversion for argument 2 from ‘const char*’ to ‘char*’
In file included from ../src/main.cc:1:0:
../../nan/nan.h: In function ‘v8::Local<v8::Object> NanNewBufferHandle(uint32_t)’:
../../nan/nan.h:675:61: error: could not convert ‘node::Buffer::New(v8::Isolate::GetCurrent(), ((size_t)size))’ from ‘v8::MaybeLocal<v8::Object>’ to ‘v8::Local<v8::Object>’
     return node::Buffer::New(v8::Isolate::GetCurrent(), size);
                                                             ^
../../nan/nan.h: In function ‘v8::Local<v8::Object> NanBufferUse(char*, uint32_t)’:
../../nan/nan.h:682:12: error: ‘Use’ is not a member of ‘node::Buffer’
     return node::Buffer::Use(v8::Isolate::GetCurrent(), data, size);
            ^
make: *** [Release/obj.target/runas/src/main.o] Ошибка 1
gyp ERR! build error 
gyp ERR! stack Error: `make` failed with exit code: 2
gyp ERR! stack     at ChildProcess.onExit (/opt/atom/resources/app/apm/node_modules/npm/node_modules/node-gyp/lib/build.js:269:23)
gyp ERR! stack     at ChildProcess.emit (events.js:98:17)
gyp ERR! stack     at Process.ChildProcess._handle.onexit (child_process.js:820:12)
gyp ERR! System Linux 3.13.0-68-generic
gyp ERR! command "node" "/opt/atom/resources/app/apm/node_modules/npm/node_modules/node-gyp/bin/node-gyp.js" "rebuild"
gyp ERR! cwd /tmp/apm-install-dir-1151016-25623-bm7lg8/node_modules/recent-files-fuzzy-finder/node_modules/pathwatcher/node_modules/runas
gyp ERR! node -v v0.10.40
gyp ERR! node-gyp -v v2.0.2
gyp ERR! not ok 
npm ERR! Linux 3.13.0-68-generic
npm ERR! argv "/opt/atom/resources/app/apm/bin/node" "/opt/atom/resources/app/apm/node_modules/npm/bin/npm-cli.js" "--globalconfig" "/home/vtikhonov/.atom/.apm/.apmrc" "--userconfig" "/home/vtikhonov/.atom/.apmrc" "install" "/tmp/d-1151016-25623-104hkge/package.tgz" "--target=0.34.0" "--arch=x64"
npm ERR! node v0.10.40
npm ERR! npm  v2.13.3
npm ERR! code ELIFECYCLE

npm ERR! runas@2.0.0 install: `node-gyp rebuild`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the runas@2.0.0 install script 'node-gyp rebuild'.
npm ERR! This is most likely a problem with the runas package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     node-gyp rebuild
npm ERR! You can get their info via:
npm ERR!     npm owner ls runas
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     /tmp/apm-install-dir-1151016-25623-bm7lg8/npm-debug.log
```

--- a/package.json
+++ b/package.json
@@ -6,11 +6,11 @@
   "repository": "https://github.com/viddo/recent-files-fuzzy-finder",
   "license": "MIT",
   "engines": {
-    "atom": ">=0.174.0 <2.0.0"
+    "atom": ">=1.2.0 <2.0.0"
   },
   "dependencies": {
     "fs-plus": "^2.6.0",
-    "pathwatcher": "^4.3.1",
+    "pathwatcher": "^6.3.0",
     "temp": "~0.8.1",
     "underscore-plus": "^1.6.6",
     "wrench": "^1.5"

